src/browser/components/molecules/TaskList/index.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react'
+import Layout from '../../molecules/Layout/index'
+import Task from '../Task/index'
+import styles from './index.css';
+const moment = require('moment')
+
+export default class TaskList extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      tasks: []
+    }
+  }
+
+  componentWillMount(){
+    this.loadTasks()
+  }
+
+  loadTasks(){
+    const fetchDetails = {
+      method: 'GET',
+      mode: 'cors',
+      headers: new Headers({
+        'Accept': 'application/json, application/xml, text/plain, text/html, *.*',
+        'Content-Type': 'application/json',
+         }),
+      credentials: 'same-origin'
+    }
+    fetch('http://noob.learnersguild.dev/api/task', fetchDetails)
+    .then( data => {
+      return data.json()
+    })
+    .then( result => {
+      this.setState({
+        tasks: result.sort((a,b) => {
```

Comment:
the sort function can be changed to `result.sort((a,b) => moment(a.due_date).isBefore(b.due_date) `